lib/api/BackbeatAPI.js

```diff
@@ -43,6 +38,8 @@ class BackbeatAPI {
         this._validSites = this._repConfig.destination.bootstrapList.map(
             item => item.site);
 
+        this._expiry = config.metrics.expiryS;
```

Comment:
Because the scope is the whole backbeat API, we should either prefix/suffix the variable name with the context (e.g. `_metricsExpiry`), or embed metrics-related vars into a `_metrics` object.